ACPs/108-evm-event-importing/README.md

```diff
@@ -0,0 +1,117 @@
+```text
+ACP: 108
+Title: EVM Event Importing Standard
+Author(s): Michael Kaplan (https://github.com/mkaplan13)
+Discussions-To: https://github.com/avalanche-foundation/ACPs/discussions/109
+Status: Proposed
+Track: Best Practices Track
+```
+
+## Abstract
+
+Defines a standard smart contract interface and abstract implementation for importing EVM events from any blockchain within Avalanche using [Avalanche Warp Messaging](https://docs.avax.network/build/cross-chain/awm/overview).
+
+## Motivation
+
+The implementation of Avalanche Warp Messaging within `coreth` and `subnet-evm` exposes a [mechanism for getting authenticated hashes of blocks](https://github.com/ava-labs/subnet-evm/blob/master/contracts/contracts/interfaces/IWarpMessenger.sol#L43) that have been accepted on blockchains within Avalanche. There is currently no clear standard for using authenticated block hashes in smart contracts within Avalanche, making it difficult to build applications that leverage this mechanism. In order to make effective use of authenticated block hashes, contracts must be provided encoded block headers that match the authenticated block hashes and also Merkle proofs that are verified against the state or receipts root contained in the block header. 
```

Comment:
nit: you may also want to link to ACP-75 (saying this is a specific instance of why they are useful)

Reply:
Good call, added now. 

Question though...ACP-75 doesn't directly support surfacing the acceptance within the VM right? Only in consensus?